### PR TITLE
Nessie client doc improvements

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: clients/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java [hunk 1]
```diff
@@ -76,17 +81,18 @@ public final class NessieConfigConstants {
...
   /**
-   * Config property name ("{@value #CONF_READ_TIMEOUT}") for the network transport read-timeout.
+   * Config property name ({@value #CONF_READ_TIMEOUT}) for the network transport read-timeout,
+   * default is {@value #DEFAULT_READ_TIMEOUT_MILLIS}.
    */
   public static final String CONF_READ_TIMEOUT = "nessie.transport.read-timeout";
   /**
-   * Config property name ("{@value #CONF_CONNECT_TIMEOUT}") for the network transport connect
-   * timeout.
+   * Config property name ({@value #CONF_CONNECT_TIMEOUT}) for the network transport connect timeout
```
Comment (ajantha-bhat) [inline clients/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java:89]: nit: 
```suggestion
   * Config property name ({@value #CONF_CONNECT_TIMEOUT}) for the network transport connection timeout
```

**Review by ajantha-bhat — APPROVED** [review]

File: clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java [hunk 2]
```diff
@@ -58,9 +59,12 @@ class Builder {
     private SSLContext sslContext;
     private SSLParameters sslParameters;
     private int readTimeoutMillis =
-        Integer.parseInt(System.getProperty("sun.net.client.defaultReadTimeout", "25000"));
+        Integer.parseInt(
```
Comment (ajantha-bhat) [inline clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java:62]: I think we missed `System.getProperty` hence the tests are failing.

Comment (ajantha-bhat) [reply clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java:62]: `DEFAULT_READ_TIMEOUT_MILLIS` and `DEFAULT_CONNECT_TIMEOUT_MILLIS` can be a string.

Comment (dimas-b) [reply clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java:62]: Maybe `getInteger`?

**Review by dimas-b — DISMISSED** [review]

**Review by ajantha-bhat — DISMISSED** [review]

**Review by dimas-b — APPROVED** [review]

**Pull request merged**